src/core/loopback-setup.c

```diff
@@ -201,7 +203,7 @@ int loopback_setup(void) {
                 return log_error_errno(r, "Failed to enqueue loopback interface start request: %m");
 
         while (state.n_messages > 0) {
-                r = sd_netlink_wait(rtnl, USEC_INFINITY);
+                r = sd_netlink_wait(rtnl, LOOPBACK_SETUP_TIMEOUT_USEC);
```

Comment:
hmm, the check_loopback() call also has an USEC_INFINITY wait in it, so it should be fixed too I figure